webapp/src/component/App.tsx

```diff
@@ -96,7 +58,12 @@ export class App extends React.Component {
     globalContext.actions?.setGlobalError(error as GlobalError);
     throw error;
   }
+
   render() {
+    const {
+      billing: { GlobalLimitPopover },
+    } = getEe();
```

Comment:
Can't we just do `import {GlobalLimitPopover} from tg.ee` or something like that?